apps/TakeInitiative.Bestiary/Program.cs

```
@@ -23,47 +24,28 @@ private static async Task Main(string[] args)
             Debug.WriteLine("Loading dev configuration");
             configBuilder = configBuilder.AddJsonFile("appsettings.development.json", optional: true);
         }
+        
 
-        var connstring = builder.Configuration.GetConnectionString("BestiaryDB") ?? throw new OperationCanceledException("Required Config 'ConnectionStrings:BestiaryDB' is missing.");
+        var LiteDB_Path = builder.Configuration.GetConnectionString("Bestiary_LiteDB");
+        if (LiteDB_Path == null || LiteDB_Path == "") {
+            LiteDB_Path = "Filename=BestiaryDB.db"; // Default path if not set in config -  just in the same folder
+        }

```

nitpick: I appreciate defaults, but I think its better to throw an exception here, as it is a fundamental mistake of the user if they have not properly configured the app.